frontend/src/actions/users/get_approval_key.js

```diff
@@ -0,0 +1,49 @@
+import { get } from '../../lib/api';
+import * as constants from '../../constants/users';
+
+/**
+ * Get approval key for paypal
+ */
+
+export default (userid, options={}) => {
+  // Redirect to homepage that will confirm the preapproval key
+  const callback = `${options.baseUrl}/?paypalApprovalStatus=`;
+  const preapprovalTemplate = '${preapprovalKey}';
```

Comment:
We don't know the preapproval key at this point in the sequence, paypal fills it out and it needs that format.